### PR TITLE
fixed: capitalised all input fields

### PR DESCRIPTION
**Pull request by steph237**

### CODE REVIEW

File: src/pages/lab-results.js [hunk 1]
```diff
@@ -44,6 +44,16 @@ const LabResults = () => {
...
+Array.prototype.forEach.call(
+    document.querySelectorAll("input[type=text]"),
+    function (input) {
+      input.addEventListener("keyup", function () {
+        console.log("Key upped!");
+        input.value = input.value.toUpperCase();
+      });
+    }
+  );
+
```
Comment (nSedrickm) [inline src/pages/lab-results.js:56]: Same here, as mentioned for the follow ups page this code is better run in a useEffect hook

The text inputs on this page are not capitalized aswell

File: src/pages/new-record.js [hunk 2]
```diff
@@ -36,6 +36,16 @@ const NewRecord = () => {
...
+  Array.prototype.forEach.call(
+    document.querySelectorAll("input[type=text]"),
+    function (input) {
+      input.addEventListener("keyup", function () {
+        console.log("Key upped!");
+        input.value = input.value.toUpperCase();
+      });
+    }
+  );
+
```
Comment (nSedrickm) [inline src/pages/new-record.js:48]: Text inputs are capitalized on typing. Well done!

However, it only works when the page is first visited. Creating a second record does not have the intented result.

Perhaps the useEffect hook could fix that

File: src/pages/specimen-collection.js [hunk 3]
```diff
@@ -43,6 +43,16 @@ const SpecimenCollection = () => {
...
+  Array.prototype.forEach.call(
+    document.querySelectorAll("input[type=text]"),
+    function (input) {
+      input.addEventListener("keyup", function () {
+        console.log("Key upped!");
+        input.value = input.value.toUpperCase();
+      });
+    }
+  );
+
```
Comment (nSedrickm) [inline src/pages/specimen-collection.js:55]: As per the other comments. Some updates  here aswell

**Review by nSedrickm — CHANGES_REQUESTED** [review]
Great progress here! left some comments to help improve

I noticed only the TextFields were referenced for capitalization. What of the radios and checkbox input fields? Their values were to be capitalized aswell. Will these be handled here or in another PR?

**Comment by steph237** [comment]
> Great progress here! left some comments to help improve
> 
> I noticed only the TextFields were referenced for capitalization. What of the radios and checkbox input fields? Their values were to be capitalized aswell. Will these be handled here or in another PR?

Yes as of now only the text fields are capitalized. Still figuring out if the radio buttons will be capitalized on clicked or if all the options will be capitalised

**Pull request closed without merging**